docs/source/checks/tabular/data_integrity/plot_class_imbalance.py

```diff
@@ -3,32 +3,32 @@
 .. _plot_tabular_class_imbalance:
 
 Class Imbalance
-***************
+****************
```

Comment:
```suggestion
***************
```
Why add another `*`? It should match the length of the title, and it does.